portlets/src/main/webapp/vue-app/rule/components/ManageRules.vue

```diff
@@ -157,10 +157,10 @@ export default {
     },
     updateRule(ruleDTO) {
       this.$RuleServices.updateRule(ruleDTO)
-        .then(rule => {
+        .then(() => {
           this.addSuccess = true;
           this.updateMessage = 'updated';
-          this.rules.push(rule);
+          this.rules =  this.getRules();
```

Comment:
When updating we need to just replace the rule with the updated one and not reload all rules right ?

Reply:
done!